### DISCUSSION TITLE
Datahub query a Block Item inside of a Block Item

### DISCUSSION THREAD

**Discussion by DerickVanDerSandt** [pimcore/pimcore#9264] [Q&A / Support]
Hi, can anyone please advise me on the syntax to use to query an object that has a Block and another block inside of it example:

block1{
   name
   block2{
      type
   }
}

This is my data type - https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Data_Types/Blocks.html

I am able to query the first block but I am unable to access the second block. I know when telling Datahub which fields to access you cannot select child items for a block as it provides access to everything inside the block except the second block. The fields of the second block do appear on the IntelliSense but always returns null

**Comment by weisswurstkanone** [comment]
Nesting blocks inside blocks is not supported as the mentioned documentation says.

**Reply by DerickVanDerSandt** [reply]
Hi, Pimcore is currently in the process of creating a Pimcore System for us and did the following - 
![image](https://user-images.githubusercontent.com/62931831/119985200-a3f16900-bfc2-11eb-8aab-1630a0e36ffe.png)

**Comment by weisswurstkanone** [comment]
Shouldn't be allowed. I would say then we need a similar check like https://github.com/pimcore/pimcore/pull/8759 wdyt ?
